src/test/java/uk/gov/hmcts/dts/fact/controllers/admin/list/AdminAreasOfLawControllerTest.java

```diff
@@ -47,4 +44,135 @@ void shouldReturnAllAreasOfLaw() throws Exception {
             .andExpect(status().isOk())
             .andExpect(content().json(allAreasOfLawJson));
     }
+
+    @Test
+    void shouldReturnAnAreaOfLaw() throws Exception {
+        final AreaOfLaw areaOfLaw = getAreasOfLaw().get(0);
+        when(adminAreasOfLawService.getAreaOfLaw(100)).thenReturn(areaOfLaw);
+
+        final String areaOfLawJson = new ObjectMapper().writeValueAsString(areaOfLaw);
```

Comment:
The construction for ObjectMapper is called numerous times in this class. Would be better to define a constant class variable at the top and just use that in all the tests:
private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();